packages/office-ui-fabric-react/src/components/pickers/BasePicker.test.tsx

```diff
@@ -328,4 +329,183 @@ describe('BasePicker', () => {
     const currentPicker = picker.current!.items;
     expect(currentPicker).toHaveLength(0);
   });
+
+  it('Closes menu when escape is pressed', () => {
+    jest.useFakeTimers();
+    document.body.appendChild(root);
+
+    const picker = React.createRef<IBasePicker<ISimple>>();
+
+    ReactDOM.render(
+      <BasePickerWithType
+        onResolveSuggestions={onResolveSuggestions}
+        onRenderItem={onRenderItem}
+        onRenderSuggestionsItem={basicSuggestionRenderer}
+        componentRef={picker}
+      />,
+      root,
+    );
+
+    const input = document.querySelector('.ms-BasePicker-input') as HTMLInputElement;
+    input.focus();
+    input.value = 'asdff';
+    ReactTestUtils.Simulate.input(input);
+    jest.runAllTimers();
+
+    const suggestions = document.querySelector('.ms-Suggestions') as HTMLElement;
+    expect(suggestions).toBeTruthy();
+
+    ReactTestUtils.Simulate.keyDown(input, { which: 27 });
+    const againSugs = document.querySelector('.ms-Suggestions') as HTMLElement;
```

Comment:
I'm very cautious about DRYing tests, but in this case it feels like it might make sense to extract this to a 1-liner called `getSuggestionsElement`, since it happens a lot and involves an ugly cast.